src/xrGame/console_commands.cpp

```diff
@@ -181,6 +181,33 @@ class CCC_MemStats : public IConsole_Command
 };
 
 // console commands
+class CCC_Spawn_to_inv : public IConsole_Command
+{
+public:
+    CCC_Spawn_to_inv(LPCSTR N) : IConsole_Command(N){};
+    virtual void Execute(LPCSTR args)
```

Comment:
better: `void Execute(LPCSTR args) override`

Reply:
Well, I'll take this into account and change it to new methods